File: Arrays/InsertEnd/ArrayInsertEnd.js
function insertAtEnd(arr,n,key,size)
{
    if(n>=size)
    {
        return n;
    }
    arr[n]=key;
    return (n+1);
}
let arr = new Array(20);
    arr[0] = 12;
    arr[1] = 16;
    arr[2] = 20;
    arr[3] = 40;
    arr[4] = 50;
    arr[5] = 70;
    let capacity = 20;
    let n = 6;
    let i, key = 26;

    console.log("Before Insertion: ");
    for (i = 0; i < n; i++)
      console.log(arr[i]+" ");
    console.log("</br>");

    // Inserting key
    n = insertAtEnd(arr, n, key, capacity);

    console.log("After Insertion: ");
    for (i = 0; i < n; i++)
      console.log(arr[i]+" ");

File: Arrays/InsertEnd/ArrayInsertEnd.c
#include <stdio.h>

int insertAtEnd(int arr[], int key, int n, int size)
{
    if (n >= size)
    {
        return n;
    }
    arr[n] = key;
    return (n + 1);
}

int main()
{
    int arr[6] = {12, 3, 34, 35, 5}; 
    int size = sizeof(arr) / sizeof(arr[0]);
    int n = 5; 
    int i, key = 26;

    printf("\nBefore Insertion: ");
    for (i = 0; i < n; i++)
    {
        printf("%d  ", arr[i]);
    }

    n = insertAtEnd(arr, key, n, size);

    printf("\nAfter Insertion: ");
    for (i = 0; i < n; i++)
    {
        printf("%d  ", arr[i]);
    }

    return 0;
}